karma.conf.js

```diff
@@ -37,7 +37,8 @@ module.exports = function(config) {
       '**/*.js': ['sourcemap']
     },
     frameworks: ['mocha', 'chai'],
-    files: ['amplitude-snippet.min.js', 'build/snippet-tests.js', 'build/tests.js'],
+    // files: ['amplitude-snippet.min.js', 'build/snippet-tests.js', 'build/tests.js'], @TODO: Fix flaky build/snippet-tests.js and re-enable
```

Comment:
Should we remove this commented code?

Reply:
IMO leave it as a todo until we can fix then re-enable the flaky snippet tests